CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+*6.0.4*
```

Comment:
```suggestion
**6.0.4**
```

Reply:
Funny, my vim was hiding the double stars but not on the line itself. So they looked similar to me…